airgun/views/login.py

```diff
@@ -7,7 +7,7 @@
 class LoginView(View, ClickableMixin):
     username = TextInput(id="login_login")
     password = TextInput(id="login_password")
-    login_text = Text('//*[@id="login-footer-text"]')
+    login_text = Text(".//footer[contains(@class,'login-pf-page-footer')]")
     logo = Text('//img[@alt="logo"]')
     submit = Text('//button[@type="submit"]')
     version = Text('//*[@id="version"]')
```

Comment:
Any reason not to delete `version` if you stopped using it and there isn't any element with this id?

Reply:
@peterdragun Version is removed from UI and it is moved as a part of footer so we are good to remove this @adarshdubey-star

Reply:
Yes, I was going to remove it just wanted to have an ack to remove it. @peterdragun